client/src/firebase/firebaseConfig.js

```diff
@@ -14,3 +14,4 @@ const firebaseConfig = {
 // Initialize Firebase
 export const app = initializeApp(firebaseConfig);
 export const analytics = getAnalytics();
+setUserProperties(analytics, { favorite_food: 'apples' });
```

Comment:
whats this

Reply:
just for testing stuff and forgot to delete it